styles/style.css

```diff
@@ -108,7 +111,15 @@ content {
     font-weight: 700;
 }
 
+.description__sentences {
+    font-size: 20px;
+}
+.description__sentences > a {
+    color: #01c1e9;
```

Comment:
This blue color didn't fit the rest of ui. Greenish color would fit better like in logo.